configuration/external_parameters.rst

```diff
@@ -453,6 +453,18 @@ Symfony provides the following env var processors:
                 'auth' => '%env(file:AUTH_FILE)%',
             ));
 
+``env(key:FOO:BAR)``
+    Retrieves key ``FOO`` from array value ``BAR``:
+
+    .. code-block:: yaml
+
+        parameters:
+            env(APP_SECRETS): "{\"database_password\": \"secret\"}"
+            database_password: '%env(key:database_password:APP_SECRETS)%'
```

Comment:
Don't you need the `json` processor before using the `key` processor?

Reply:
You're completely right, it's updated now!